src/proto/srv/srv.proto

```diff
@@ -12,12 +12,13 @@ package srv;
 option go_package = "github.com/daos-stack/daos/src/control/common/proto/srv";
 
 message NotifyReadyReq {
-	string uri = 1; // CaRT URI
-	uint32 nctxs = 2; // Number of CaRT contexts
+	string uri = 1; // Primary CaRT URI
+	uint32 nctxs = 2; // Number of primary CaRT contexts
```

Comment:
I don't really understand what a CaRT context is, so this may be a dumb question, but do we not need to know this information for the secondary URIs?

Reply:
It doesn't look like we actually use the existing nctxs parameter in this message anywhere in the codebase, aside from filling it out. If we find we need to present it to clients, I will happily add one for the secondary ctxs.

Reply:
Eventually we will need to know number of secondary contexts in cart on the client side.
Right now on a secondary provider cart always sends all RPCs to tag=0; once we add a new API to specify number of server tags to the client cart will then randomize which server tag to send to  (rand() % n_secondary_ctxs)

Reply:
Yeah, I can add it in a follow-on.